dns-controller/pkg/dns/dnscontroller.go

```diff
@@ -32,6 +32,8 @@ import (
 	"k8s.io/kubernetes/federation/pkg/dnsprovider/rrstype"
 )
 
+const DefaultTTL = time.Minute
```

Comment:
One of these days we are going to have a fabulous configuration something or other for this stuff - minute sounds good though.. but I know plenty of DNS fans who would have something to say about this :)